src/fideslang/models.py

```diff
@@ -879,12 +871,22 @@ class PrivacyDeclaration(BaseModel):
         default_factory=list,
         description="The categories of personal data that this system shares with third parties.",
     )
-    cookies: Optional[List[Cookies]] = Field(
+    cookies: Optional[Any] = Field(  # type: ignore
         default=None,
-        description="Cookies associated with this data use to deliver services and functionality",
+        description="Deprecated, do not use.",
+        deprecated=True,
     )
     model_config = ConfigDict(from_attributes=True)
 
+    @field_validator("cookies")
+    @classmethod
+    def validate_cookies(cls, value: Optional[Any]) -> None:  # type: ignore
+        """
+        Validate that the `cookies` field is deprecated and warn that it should not be used.
+        """
+        if value is not None:
+            warn("The 'cookies' field is deprecated and should not be used.")
```

Comment:
may be nice just to make it crystal clear that `cookies` values will be ignored?

Reply:
👍  added here https://github.com/ethyca/fideslang/pull/27/commits/956b6ec6fefa4fd66e686878317c94fa25b55e15#diff-b0045e1de80641317fa3047363156dc405a70a28ea7fa075955d1072ccc9a151R1117